tools/.github-cla-signers

```diff
@@ -18,6 +18,7 @@ bmhughes
 candlerb
 cawamata
 cclauss
+chifac08
```

Comment:
I confirmed this user has signed the CLA. Welcome to cloud-init @chifac08 and thanks!